policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/education_property_tax_credit/age_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Manitoba provides different tax credits according to different age groups.
```

Comment:
description: Manitoba provides the basic tax credits for filters at or above this age threshold.